snap/info.go

```diff
@@ -2099,3 +2099,16 @@ func RegistryPlugAttrs(plug *PlugInfo) (account, registry, view string, err erro
 
 	return account, registry, view, nil
 }
+
+// RefreshFailures holds information about snap failed refreshes.
+type RefreshFailuresInfo struct {
+	// ToRevision is the target revision that caused the refresh failure.
+	ToRevision Revision `json:"to-revision,omitempty"`
```

Comment:
I wonder if the To/to- here adds to clarity actually or not

Reply:
I don't have strong opinions on this but I feel `ToRevision` is clearer. @zyga suggested it in the spec, maybe there is better reason?

Reply:
we usually call this the refresh revision, refresh is in the struct name, and revision is the field here

Reply:
renamed